README.md

```diff
@@ -105,3 +140,15 @@ Everywhere:
   * Will there be a hard deprecation header? or just an error?
   * How do we deal with non-JSON responses (like just an integer)
   * Do we handle team names yet? How so? Versions?
+
+# SDK Supported Features
+
+| SDK        | [Invoke](https://docs.readme.build/docs/getting-started-2) | Private Services | [Files](https://docs.readme.build/v1.0/docs/sending-files)   | Outputs |
+| ---------- | :----: | :--------------: | :-----: | :-----: |
+| [**Node**](https://github.com/readmeio/api) |✔|✔|✔|✔|
+| [**Python**](https://github.com/readmeio/api-python) |✔|✔|✕|✕|
+| [**Ruby**](https://github.com/readmeio/api-ruby) |✔|✔|✕|✕|
+| **Slack**  |✔|✔|✕|✕|
```

Comment:
Can you link to a docs page for slack and cron? Even if we dont have any content yet.